h/static/scripts/annotator/anchoring/types.coffee

```diff
@@ -0,0 +1,55 @@
+Annotator = require('annotator')
+$ = Annotator.$
+xpathRange = Annotator.Range
+
+
+# Helper function for throwing common errors
+missingParameter = (name) ->
+  throw new Error('missing required parameter "' + name + '"')
+
+
+###*
+# class:: RangeAnchor(range)
+#
+# This anchor type represents a DOM Range.
+#
+# :param Range range: A range describing the anchor.
+###
+class RangeAnchor
```

Comment:
This class isn't broken out into a separate library. I'm actually not sure how useful it is in practice (the position anchors perform very well) and it doesn't map straightforwardly to any Web Annotations Data Model selector types, so I'm holding it here for the moment.